x/merkledb/proof.go

```diff
@@ -840,9 +837,9 @@ func addPathInfo(
 			if existingChild, ok := n.children[index]; ok {
 				compressedKey = existingChild.compressedKey
 			}
-			childPath := key.Extend(ToToken(index, t.tokenSize), compressedKey)
-			if (shouldInsertLeftChildren && childPath.Less(insertChildrenLessThan.Value())) ||
-				(shouldInsertRightChildren && childPath.Greater(insertChildrenGreaterThan.Value())) {
+			childKey := key.Extend(ToToken(index, t.tokenSize), compressedKey)
+			if (shouldInsertLeftChildren && childKey.Less(insertChildrenLessThan.Value())) ||
+				(shouldInsertRightChildren && childKey.Greater(insertChildrenGreaterThan.Value())) {
```

Comment:
If we pulled this rename into it's own PR I think we could merge it rq. Doesn't seem related to the root change.

Reply:
https://github.com/ava-labs/avalanchego/pull/2267